.changeset/rude-geckos-happen.md

```diff
@@ -0,0 +1,7 @@
+---
+"@manypkg/get-packages": patch
+"@manypkg/tools": patch
+"@manypkg/cli": minor
+---
+
+Add `knip` and fix all issues raised by it.
```

Comment:
Adding `knip` is an internal change and the users (changesets target audience) don't care about this. Don't bother documenting what changes internally but please document the user-facing changes using changesets (like for example the removal of `find-up`)